pkg/ebpf/c/tracee.bpf.c

```diff
@@ -543,8 +543,6 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
     // Update the process tree map (filter related) if the parent has an entry.
 
     policies_config_t *policies_cfg = get_policies_config(&p);
```

Comment:
since `get_policies_config()` function is now one line, I think it will be simpler to just put it inline